internal/lib/lib.go

```diff
@@ -1413,6 +1413,8 @@ func InformersToRegister(kclient *kubernetes.Clientset, oclient *oshiftclient.Cl
 	// AKO must watch over Pods in case of NodePortLocal, to get Antrea annotation values.
 	if GetServiceType() == NodePortLocal {
 		allInformers = append(allInformers, utils.PodInformer)
+	} else {
```

Comment:
This is not required as we have already appended endpointInformer. We should have if..elseif..else loop here. I think this happened after merge of endpointslice PR.

Reply:
fixed